apps/blockchain/test/blockchain/transaction_test.exs

```diff
@@ -3,12 +3,10 @@ defmodule Blockchain.TransactionTest do
   use EthCommonTest.Harness
   doctest Blockchain.Transaction
 
-  require Logger
-
   alias Blockchain.Transaction
   alias Blockchain.Transaction.Signature
 
-  @chains ~w(
+  @forks ~w(
```

Comment:
I changed this to `@forks` but I'm not sure if it's better or not. It seems like there's no clear way to express this. They're technically the name of the chain after a fork, so maybe `@chains` was better? I also thought about `networks` but that too seems to mean something else (like a test network).

Reply:
I like `@forks` better too. 👍